contracts/optimistic-ethereum/OVM/verification/OVM_FraudVerifier.sol

```diff
@@ -122,10 +122,10 @@ contract OVM_FraudVerifier is Lib_AddressResolver, OVM_FraudContributor, iOVM_Fr
         );
 
         require (
-            _preStateRootBatchHeader.prevTotalElements + _preStateRootProof.index == _transactionBatchHeader.prevTotalElements + _transactionProof.index,
+            _preStateRootBatchHeader.prevTotalElements + _preStateRootProof.index + 1 == _transactionBatchHeader.prevTotalElements + _transactionProof.index,
```

Comment:
`preStateRoot` for a given `transaction` sits at `transaction.index - 1`, not at `transaction.index`.